.github/workflows/qpext-docker-publish.yml

```
@@ -71,9 +76,14 @@ jobs:
           # Use Docker `latest` tag convention
           [ "$VERSION" == "master" ] && VERSION=latest
 
-          echo IMAGE_ID=$IMAGE_ID
-          echo VERSION=$VERSION
-
-          docker tag $IMAGE_NAME $IMAGE_ID:$VERSION
-          docker push $IMAGE_ID:$VERSION
-
+          echo VERSION=$VERSION >> $GITHUB_ENV
+          echo IMAGE_ID=$IMAGE_ID >> $GITHUB_ENV
+
+      - name: Build and push
+        uses: docker/build-push-action@v3
+        with:
+          platforms: linux/amd64,linux/ppc64le
```

```suggestion
          platforms: linux/amd64,linux/arm/v7,linux/arm64/v8,linux/ppc64le,linux/s390x
```

could you please include these architecutres? for feature parity with knative serving:

```
$ docker run --rm mplatform/mquery gcr.io/knative-releases/knative.dev/serving/cmd/activator@sha256:9cb7c468f0975079c7b86d1b2b6b61dd9b5e64d38cd92ccb5239348ce5bd9dee
Image: gcr.io/knative-releases/knative.dev/serving/cmd/activator@sha256:9cb7c468f0975079c7b86d1b2b6b61dd9b5e64d38cd92ccb5239348ce5bd9dee (digest: sha256:9cb7c468f0975079c7b86d1b2b6b61dd9b5e64d38cd92ccb5239348ce5bd9dee)
 * Manifest List: Yes (Image type: application/vnd.docker.distribution.manifest.list.v2+json)
 * Supported platforms:
   - linux/amd64
   - linux/arm
   - linux/arm64
   - linux/ppc64le
   - linux/s390x
```